docs: added troubleshooting info. about xfs filesystem (#448)
openebs/website: docs/main/troubleshooting/troubleshooting-local-storage.md

### How to fetch the OpenEBS Dynamic Local Provisioner logs?

**Workaround**

Review the logs of the OpenEBS Local PV provisioner. OpenEBS Dynamic Local Provisioner logs can be fetched using:

```
kubectl logs -n openebs -l openebs.io/component-name=openebs-localpv-provisioner
```

### Unable to mount `xfs` File System

> `ext4` PVC was successfully created and mounted to desired pod but `xfs` is failing to mount.

Reviewer: typo - `xfs`

> `ext4` PVC was successfully created and mounted to desired pod but `xfs` is failing to mount.

Reviewer: ```suggestion
The volume is created but `xfs` is failing to mount.
```

### How to fetch the OpenEBS Dynamic Local Provisioner logs?

**Workaround**

Review the logs of the OpenEBS Local PV provisioner. OpenEBS Dynamic Local Provisioner logs can be fetched using:

```
kubectl logs -n openebs -l openebs.io/component-name=openebs-localpv-provisioner
```

> ### Unable to mount `xfs` File System

Reviewer 2: Don't we need similar thing for the replicated storage?
@avishnu @dsharma-dc

Reviewer: Fair point, I'd think yes we need. I wonder why it's never reported with Mayastor though(or has it?) while lvm has so many reports.

Reviewer 2: Not sure, maybe users are already be on higher kernel versions?
Also also it's probably "worse" on mayastor as I think the xfs utils may be even newer. Also more problematic there because volumes can move between nodes! :(
